decentraland/unity-renderer
# feat: added new asset bundle cdn flow behind a feature flag

## What does this PR change?

Closes #4243 

The new Asset bundle pipeline has a new endpoint where we can get the scene manifest to check if its assets are converted.
A new web request was added to fill the `ContentProvider` of that scene with information about the asset bundles.
Example: `https://ab-cdn.decentraland.org/manifest/QmZjgYWWsFKsVCa6bbTQ9BDs2KBzgpmcLxHj92F9qCAouH.json`


## How to test the changes?

Enter this link:
https://play.decentraland.org/?explorer-branch=feat/ab-new-cdn&ASSET_BUNDLES_DOMAIN=https://ab-cdn.decentraland.org&ENABLE_AB-NEW-CDN

- use `/detectabs` to check that most popular scenes are green, some might not have been converted yet


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
